keyboards/aki27/cocot46plus/keyboard.json

```diff
@@ -0,0 +1,130 @@
+{
+  "keyboard_name": "cocot46plus",
+  "manufacturer": "aki27",
+  "maintainer": "markstos",
+  "tags": ["unibody","ergo","corne","trackball","encoder","rgb"],
+  "usb": {
+    "device_version": "1.0.0",
+    "pid": "0x0003",
+    "vid": "0x1727"
```

Comment:
This vendor ID is in use: 
https://www.the-sz.com/products/usbid/index.php?v=0x1727&p=&n=

Generally, the further past 0x4000 you are, the better.

Reply:
Thanks for catching that. I've notified the original keyboard designer and will give them a bit to respond, as they have some other keyboards with the same conflicting vendor ID. If I don't hear from them after a while, I guess we should just pick a new ID for this keyboard.

Reply:
I did not hear back from the keyboard creator after several attempts on multiple mediums, so I picked a new vendor ID myself and pushed a commit for that.